deploy/kubernetes/v1.12+/attacher.yaml

```diff
@@ -27,6 +27,9 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["csi.storage.k8s.io"]
+    resources: ["csinodeinfos"]
+    verbs: ["get", "list", "watch"]
 
```

Comment:
Since I cannot to comment there, do we still need `resource: ["events"]` at line 20? It seems not needed per: https://github.com/kubernetes-csi/external-attacher/blob/9da8c6d20d58750ee33d61d0faf0946641f50770/deploy/kubernetes/rbac.yaml

Reply:
I'm not aware about events. I'll try to remove it and see if it still works.

Reply:
Removed "events" and no problem happened. Updated.